app/src/main/java/io/intrepid/contest/base/BaseFragmentActivity.java

```diff
@@ -24,12 +24,16 @@ protected void onCreate(Bundle savedInstanceState) {
         if (savedInstanceState == null) {
             Intent intent = getIntent();
             Fragment fragment = createFragment(intent);
-            getSupportFragmentManager()
-                    .beginTransaction()
-                    .add(R.id.fragment_container, fragment)
-                    .commit();
+            hostFragment(fragment);
         }
     }
 
+    protected void hostFragment(Fragment fragment) {
```

Comment:
I was intended on reusing it in Actviity subclasses, but ended up going with the viewpager approach. In any event, the fragment transaction code isn't too bulky o boilerplaty that we have to avoid it at all costs. I'll revert this.